tests/gds_kernel_latency.c

```diff
@@ -559,12 +555,16 @@ int pp_close_ctx(struct pingpong_context *ctx)
         {
                 if( ctx->exp_send_info == 1 )
                 {
-                        free(ctx->txbufexp);
+                        free(ctx->txbufexp);                        
                         free(ctx->txbufexp_size);
                         free(ctx->txbufexp_lkey);
                         free(ctx->txbufexp_addr);
                 }
         }
+        
```

Comment:
why did you move ibv_close_device here?

Reply:
it's an oversight, I was debugging a free error